lib/plugin.js

```diff
@@ -12,7 +12,7 @@ function gtmClient(ctx, initialized) {
     },
     push(obj) {
       if (!window[_layer]) {
-        window[_layer] = [{ 'gtm.start': new Date().getTime(), event: 'gtm.js' }]
```

Comment:
This event is also being sent by the original snippet. Any reason to remove?

Reply:
With the modifications I made, the window[_layer] is always initialized, so it will never run, beside that I think that should be done as part of the init script (and it is in module.js [here](https://github.com/nuxt-community/gtm-module/blob/master/lib/module.js#L59)) and not here in the push method. But I can revert this change if you want.

Reply:
Indeed making sense.